src/components/new-request/name-check/name-check-conflicts.vue

```diff
@@ -0,0 +1,320 @@
+                  <span>
+                    <span v-if="!item.expandExtraInfo" right>View </span>
+                    <span v-else right>Hide </span>
+                    tips for how to create a unique name
+                  </span>
+                </v-btn>
+              </v-row>
+              <v-row v-if="item.expandExtraInfo" no-gutters class="pt-3">
+                <v-col>
+                  <v-row no-gutters>
+                    <b>Use words that will set your name apart, such as:</b>
+                  </v-row>
+                  <v-row no-gutters>
+                    <ul>
+                      <li>a person's full Name</li>
+                      <li>a geographical location</li>
+                      <li>a coined, made-up word, or</li>
+                      <li>an acronym</li>
+                    </ul>
+                  </v-row>
+                </v-col>
+              </v-row>
+            </v-col>
+          </v-row>
+        </v-col>
+      </v-row>
+    </template>
+  </v-data-table>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+import QuickSearchNames from '@/components/new-request/name-check/quick-search-names.vue'
+import { NameCheckItemIF } from '@/interfaces/name-check-interfaces'
+import { NameCheckErrorType, NameCheckItemType } from '@/enums'
+
+@Component({
+  components: { QuickSearchNames }
+})
+export default class NameCheckConflicts extends Vue {
+  @Prop()
+  private readonly items: Array<NameCheckItemIF>
+
+  @Emit() private clearError (value: string) { }
+  @Emit() private retry (value: string) { }
+
+  private expanded: Array<NameCheckItemIF> = []
+  private headers = [
+    { text: 'Icon', value: 'icon' },
+    { text: 'Problem', value: 'problem' },
+    { text: 'Words', value: 'words' },
+    { text: '', value: 'data-table-expand' }
+  ]
+
+  get ERROR_TYPES () {
+    return NameCheckErrorType
+  }
+  get ITEM_TYPES () {
+    return NameCheckItemType
+  }
+
+  @Watch('expanded')
+  private triggerChange () { }
+
+  @Watch('items')
+  private updateExpanded (value: Array<NameCheckItemIF>) {
+    if (value && value.length > 0) {
+      const similarNameCheck: Array<NameCheckItemType | NameCheckErrorType> = [
+        this.ITEM_TYPES.EXACT_MATCH,
+        this.ITEM_TYPES.EXACT_MATCH_XPRO,
+        this.ITEM_TYPES.SIMILAR_MATCH,
+        this.ITEM_TYPES.SIMILAR_MATCH_XPRO
+      ]
+      if (similarNameCheck.includes(value[0].info)) {
+        this.expanded.push(value[0])
+      } else {
+        // if all items are loading it is a new search so reset expanded
+        let newSearch = true
+        for (let item of value) {
+          if (item.info !== this.ITEM_TYPES.LOADING) newSearch = false
+        }
+        if (newSearch) this.expanded = []
+      }
+    } else {
+      this.expanded = []
+    }
+  }
+
+  mounted () {
+    this.updateExpanded(this.items)
+  }
```

Comment:
Super minor but we usually put the vue lifecycle hooks near the top of the script, after property definitions but before computed properties etc. No action required here, your call!

Reply:
okay yeah I was wondering about this too, I'll shift it up